### DISCUSSION TITLE
Battery charged fully :)

### DISCUSSION THREAD

**Discussion by sandridgepark** [kellerza/sunsynk#279] [General]
Hi there
I've been able to sort an automation that charges the battery when Octopus gives an intelligent charge
The issue I have is getting the battery to discharge when the intelligent charge has finished, it still uses the grid 
The battery does stop charging which is good :)

Any idea what I need to do to tell it 'now discharge the battery please!' ?
I am using Sunsynk/Deye Inverter Add-on (multi)

**Comment by Hatios** [comment]
You can do this using the timer by adjusting the SOC for the current timeslot.
Set the SOC to below the current charge % with automation and it will then drain the battery to that % before using the grid.

I use a single timer slot that is active at all times, but then have the addon manipulate the SOC % for that time slot as needed.
Note that they cascade so I believe slot 6 is the best one to use for this.

This is exactly how mine is set :)
